lib/service/src/main/java/io/atlasmap/service/AtlasService.java

```diff
@@ -78,10 +82,15 @@
 import io.swagger.annotations.ApiResponse;
 import io.swagger.annotations.ApiResponses;
 
+import static io.atlasmap.v2.MappingFileType.ADM;
```

Comment:
I'm a little confused with MappingFileType.ADM.  Not trying to be too pedantic but this stuff can be confusing :).
The "mapping file" is a JSON file that's kept in the mappings directory.  It is already decorated with an ID (i.e.
atlasmapping-UI.950612.json).  The ADM and GZ files are "catalog files" that contain the source documents that are 
being mapped as well as a pre-digested version of the mappings. The ADM also contains user-defined JARs for
custom field mappings and custom transformations.  The atlasmapping-UI.950612.json exists seperately as an efficiency so when we auto update the mappings it doesn't take too long.

Given this - I like the idea of storing the mapping and catalog data hierarchically - which does mean that the backend
server and UI must be paired - I just find the MappingFileType.ADM confusing.

Reply:
Discussed with Paul and tried to find better naming for this enum, but couldn't. We might have to revisit this as a part of #1476. Let's just leave it as-is for now.